sway/desktop/shaders/tex_external.frag

```diff
@@ -34,4 +34,8 @@ void main() {
             gl_FragColor = mix(vec4(0), gl_FragColor, smooth);
         }
     }
+
+    if (gl_FragColor.a <= 0.01){
+        discard;
+    }
```

Comment:
Why not discard if equal to 0.0?

Reply:
Whoops, my bad. Accidentally left all of those in. Shouldn't make a difference